### DISCUSSION TITLE
How to pair AM43 blind Drive motor?

### DISCUSSION THREAD

**Discussion by hugalafutro** [Koenkk/zigbee2mqtt#7274] [Devices]
Hi, I bought Tuya zigbee AM43-0.45/40-ES-EZ (TY) blind motor, which I believe is this https://www.zigbee2mqtt.io/devices/TS0601_cover.html, but for the love of everything that's holy I can't figure out how to pair it with Z2M. I've factory resetted it numerous time close to the coordinator (hold the button 3 sec till led is solid red, then hold 7 sec till led flashes 3 times blue.) to no avail.

I've found 2 issues on github with same model, but they already had it paired and were sorting issues with its operation, which is at least the point I'd like to get to.

Thanks for any suggestion!

**Comment by hugalafutro** [comment]
Guess I should've just kept trying. I kept doing the same about 10 more times and one time it finally just "caught on" and interviewed fine.

**Reply by WBoerhoop** [reply]
Try to push the down button en the set button the same time.

**Reply by migzzz1018** [reply]
i know its been 2 years but, down button and set button not working

**Reply by Ocramius** [reply]
Ended up pairing by pressing in quick sequence (and holding for ~3s) "SET" and "DOWN".

There seems to be some timing between "SET" and "DOWN" being pressed for ~3s: pressing at the "same" (where "same" depends on the firmware) doesn't really work.

In the instruction manual, the wording says:

> Press setting button and down button at the same time for 3s get into pairing mode, then click <stuff in UI we don't care about>

My setup uses:

 * Zigbee2MQTT in Home Assistant
 * Home Assistant SkyConnect
